docs/guides/universal-receiver-delegate/set-default-implementation.md

```diff
@@ -47,7 +47,7 @@ After deploying the contract, we need to set its address under the **[LSP1-Unive
 
 ```typescript title="Setting address of the URD in the storage"
 import Web3 from 'web3';
-import constants from "@lukso/lsp-smart-contracts/constants.js";
+import constants from '@lukso/lsp-smart-contracts/constants.js';
```

Comment:
We can only import the `ERC725YDataKeys` constant here, so that it simplify the code below.

```suggestion
import { ERC725YDataKeys } from '@lukso/lsp-smart-contracts/constants.js';
```

We can then remove all these references below: 
![image](https://user-images.githubusercontent.com/31145285/217219788-680c6ada-f216-4980-8398-4027c68ac906.png)

Reply:
This can be done in a separate PR if too much changes @b00ste

Reply:
done